<launch>

    <node pkg="borealis_utils" type="shift_map" name="shift_map">

        <param name="target_frame" value="odom" />

        <remap from="~input_map_cloud" to="uav1/os_cloud_node/points" />
        <remap from="~output_map_cloud" to="uav1/shifted_map" />

    </node>

</launch>